packages/wallets/src/components/WalletCard/__tests__/WalletCard.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ComponentProps } from 'react';
+import { APIProvider, useBalance } from '@deriv/api';
+import { render, screen } from '@testing-library/react';
+import WalletCard from '../WalletCard';
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useBalance: jest.fn(),
+}));
+
+describe('WalletCard', () => {
+    let mockProps: ComponentProps<typeof WalletCard> = {
+        balance: '100 USD',
+        currency: 'USD',
+        iconSize: 'lg',
+        landingCompanyName: 'Deriv',
+    };
+    beforeEach(() => {
+        jest.clearAllMocks();
+        (useBalance as jest.Mock).mockImplementation(() => ({
+            ...jest.requireActual('@deriv/api').useBalance(),
+            isLoading: true,
+        }));
+        mockProps = {
+            balance: '100 USD',
+            currency: 'USD',
+            iconSize: 'lg',
+            landingCompanyName: 'Deriv',
+        };
+    });
+
+    afterAll(() => {
+        jest.resetAllMocks();
+    });
+
+    it('should render wallet card for USD wallet', () => {
```

Comment:
(minor: naming) this name does not tell me what the tests is really about,

for example, it does tests against gradient being correct and agains currency being displayed properly,